bin/build-pouchdb.js

```diff
@@ -192,12 +192,6 @@ function doBuildNode() {
     .then(buildForNode);
 }
 
-function doBuildDev() {
-  return doAll(buildForNode, buildForBrowserify)()
```

Comment:
^ classic promise mistake; this is a function that returns a promise rather than a promise itself